mkosi/__init__.py

```diff
@@ -2142,6 +2145,33 @@ def install_openmandriva(args: CommandLineArguments, root: Path, do_run_build_sc
     disable_pam_securetty(root)
 
 
+@complete_step("Installing Gentoo")
+def install_gentoo(
+    args: CommandLineArguments,
+    root: Path,
+    do_run_build_script: bool
+) -> None:
+    from .gentoo import Gentoo
+
+    # this will fetch/fix stage3 tree and portage confgired for mkosi
+    gentoo = Gentoo(args, root, do_run_build_script)
+
+    if gentoo.pkgs_fs:
+        gentoo.invoke_emerge(args, root, pkgs=gentoo.pkgs_fs)
+
+    if not do_run_build_script and args.bootable:
+        # GENTOO_DONTMOVE: call before merging sys-kernel/dracut
```

Comment:
Just so I understand this comment: Does this mean that `configure_dracut` needs to be called before emerge is called to install dracut? If so then I'd propose
```suggestion
        # Please don't move, needs to be called before installing dracut
```